src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -235,6 +241,19 @@ const AppFrame = ({ appUrl }: Props): ReactElement => {
 
       openSignMessageModal(message, msg.data.id)
     })
+
+    communicator?.on(Methods.getChainInfo, async () => {
+      const { nativeCoin } = getNetworkInfo()
+      const chainId = parseInt(networkId, 10)
+      const network = getNetworkName()
```

Comment:
@iamacook in your chains PR, do these functions still exist?

Reply:
The current issues regarding using Redux likely affect every change I have made so far, so I can take the opportunity to start again with these changes included.

Reply:
We are aware of that and we are planning to change this in  @iamacook PR. Once this one is unblocked we will push this changed to his PR

Reply:
I think @iamacook has a selector or something that we can use

Reply:
If this gets merge prior, it would be a lot easier for me to handle. Either way, I'll be happy to help fix any merge conflicts.

Reply:
Yes, we are going to merge this tomorrow or on monday at the latest I think